projects/igniteui-angular/src/lib/date-picker/date-picker.utils.ts

```diff
@@ -5,7 +5,7 @@ import { isIE } from '../core/utils';
  *
  *@hidden
  */
-export const enum DateState {
+export enum DateState {
```

Comment:
That one is hidden, though. As in, not intended to be exposed in public API at all.
There's a bit of a problem with that as the public API exposes everything like so:
[`export * from './lib/date-picker/date-picker.utils';`](https://github.com/IgniteUI/igniteui-angular/blob/7.2.x/projects/igniteui-angular/src/public_api.ts#L96)
@SAndreeva @sboykova Should this be excluded from public exports? More over, I don't see anything in the date picker util file that looks like public API. Should the entire export be dropped?

Reply:
@hidden can be removed from enum `DateState`. In case the date picker is retemplated, the user may need some of the methods in date-picker.utils for parsing masks/formats.

Reply:
You mean the static methods of the `DatePickerUtil` class that is also hidden?
I see a lot of `any`-s and types using the other non-exposed enums in there. IMHO, unless the need arises by a customer, this should probably remain unexposed.